config/environments/production.rb

```diff
@@ -1,5 +1,8 @@
 require "active_support/core_ext/integer/time"
 
+# Heroku, allowing host
+Rails.application.config.hosts << "shrouded-stream-13956.herokuapp.com"
```

Comment:
Please, sync up with Stoyan Stoyanov because he uses https://test-car.herokuapp.com. At this point I think we've created several heroku apps, so we need to agree on one